src/pixbuf.rs

```diff
@@ -106,8 +106,7 @@ impl Pixbuf {
         }
     }
 
-    pub fn new_from_stream_async<'a, P: IsA<gio::InputStream>, Q: Into<Option<&'a gio::Cancellable>>, R: FnOnce(Result<Pixbuf, Error>) + Send + 'static>(stream: &P, cancellable: Q, callback: R) {
-        let cancellable = cancellable.into();
+    pub fn new_from_stream_async<'a, P: IsA<gio::InputStream>, R: FnOnce(Result<Pixbuf, Error>) + Send + 'static>(stream: &P, cancellable: Option<&gio::Cancellable>, callback: R) {
```

Comment:
This should be `Option<IsA<gio::Cancellable>`

Reply:
Obviously everywhere not just in this single function :)

Reply:
I had hopes... :-.